gym_ai2thor/envs/ai2thor_env.py

```diff
@@ -87,12 +101,13 @@ def step(self, action, verbose=True):
         action_str = self.action_names[action]
         visible_objects = [obj for obj in self.event.metadata['objects'] if obj['visible']]
 
+        # if/else statements below for dealing with up to 13 actions
         if action_str.endswith('Object'):  # All interactions end with 'Object'
             # Interaction actions
             interaction_obj, distance = None, float('inf')
             inventory_before = self.event.metadata['inventoryObjects'][0]['objectType'] \
                 if self.event.metadata['inventoryObjects'] else []
-            if action_str == 'PutObject':
+            if action_str.startswith('Put'):
                 closest_receptacle = None
                 for obj in visible_objects:
                     # look for closest receptacle to put object from inventory
```

Comment:
put the logic of the if in a variable as we discussed

Reply:
reminder for myself to remove factory